### PR TITLE
app doc updates

### PR DESCRIPTION
**Pull request by mclark58**
Edited display.yaml to reflect changes in documentation.

### CODE REVIEW

File: ui/narrative/methods/view_expression_interactive_heatmap/display.yaml [hunk 1]
```diff
@@ -34,16 +34,15 @@ suggestions :
...
         short-hint : |
-           Expression matrix 
-        long-hint  : |
-           Expression matrix 
+           Select the ExpressionMatrix to visualize.
 
 
 description : |
-    View heatmap 
+  <p><p>This is a visualization App to view and explore an ExpressionMatrix as a heatmap. Features are displayed as rows, and the conditions as columns. Mouse over the heatmap cells to view the expression value for a given feature and a condition.</p>
```
Comment (briehl) [inline ui/narrative/methods/view_expression_interactive_heatmap/display.yaml]: Got a double-`<p>` here. Should only need one, I think. The extra one might add some extraneous padding?

**Review by briehl — CHANGES_REQUESTED** [review]

File: ui/narrative/methods/view_expression_interactive_heatmap/display.yaml (hunk 1 above)
Comment (mclark58) [reply ui/narrative/methods/view_expression_interactive_heatmap/display.yaml]: Oops! That was just a copy/paste error on my part. Sorry!

Comment (mclark58) [reply ui/narrative/methods/view_expression_interactive_heatmap/display.yaml]: I can fix that real quick

**Review by briehl — APPROVED** [review]
:+1:

**Pull request merged**